config.go

```diff
@@ -192,19 +188,95 @@ func loadConfiguration(configPath string) (oci.RuntimeConfig, ShimConfig, error)
 
 			config.ProxyType = vc.CCProxyType
 			config.ProxyConfig = pConfig
+
+			break
+		}
+	}
+
+	for k, agent := range tomlConf.Agent {
+		switch k {
+		case hyperstartAgent:
+			path := filepath.Join(agent.pauseBundlePath(),
+				pauseBinRelativePath)
```

Comment:
But this is specific to an OCI bundle, thus I don't see the point in following those conventions. And this rootfs value won't come from a config.json because it won't come from Docker for instance. This is specific to our use of hyperstart. What do you suggest to retrieve this value from somewhere ?
@sameo any suggestion on this one ?